app/controllers/hourglass/api_base_controller.rb

```diff
@@ -112,5 +112,13 @@ def book_allowed?
     def booking_forbidden_message
       t("hourglass.api.#{controller_name}.errors.booking_forbidden")
     end
+
+    def flash_array(type, messages)
+      flash[type] = content_tag :ul do
+        messages.each do |msg|
+          concat content_tag(:li, msg.html_safe)
```

Comment:
I am pretty sure you can't give an array as content for a content tag, but i haven't tried.